internal/metamorph/store/postgresql/postgres.go

```diff
@@ -375,6 +385,23 @@ func (p *PostgreSQL) SetBulk(ctx context.Context, data []*store.StoreData) error
 	return nil
 }
 
+func prepareCallbacksForSaving(callbacks []store.StoreCallback) ([]byte, error) {
+	callbacksBytes, err := json.Marshal(callbacks)
+	if err != nil {
+		return nil, err
+	}
+	return callbacksBytes, nil
+}
+
+func readCallbacksFromDB(callbacks string) ([]store.StoreCallback, error) {
+	var callbacksData []store.StoreCallback
+	err := json.Unmarshal([]byte(callbacks), &callbacksData)
```

Comment:
Then here, you wouldn't have to retype the string to []byte for unmarshalling, you could just do this:
```suggestion
func readCallbacksFromDB(callbacks []byte) ([]store.StoreCallback, error) {
	var callbacksData []store.StoreCallback
	err := json.Unmarshal(callbacks, &callbacksData)
```
But I may be wrong so please verify :)

Reply:
Done